app/controllers/sessions_controller.rb

```diff
@@ -138,6 +136,13 @@ def find_user
     User.authenticate(who, password) if password.is_a?(String) && who
   end
 
+  def validate_mfa_user
+    @user = User.find_by(id: session[:mfa_user]) if session[:mfa_user]
```

Comment:
I would expect this assignment in `find_` method. What about to split this into find and validate methods or rename to `find_and_validate_...` one?

Reply:
Yeah, sounds good. Updated.